### PR TITLE
add jacky and liqun to archinfra approvers

### PR DESCRIPTION
**Pull request by liqunfu**
Signed-off-by: Liqun Fu <liqfu@microsoft.com>

### CODE REVIEW

File: CONTRIBUTORS [hunk 1]
```diff
@@ -39,6 +39,8 @@ SIG-archinfra:
         - gramalingam
         - linkerzhang
         - prasanthpul
+        - jcwchen
```
Comment (garymm) [inline CONTRIBUTORS:42]: Please remove from contributors lists above

File: CONTRIBUTORS [hunk 2]
```diff
@@ -39,6 +39,8 @@ SIG-archinfra:
...
+        - liqunfu
```
Comment (garymm) [inline CONTRIBUTORS:43]: please keep sorted alphabetically

**Review by garymm — CHANGES_REQUESTED** [review]

**Review by garymm — APPROVED** [review]

**Comment by liqunfu** [comment]
@prasanthpul , please approve this if it is ok.

**Review by askhade — APPROVED** [review]

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM

**Pull request merged**